docs/ocis/development/testing.md

```diff
@@ -203,16 +234,16 @@ If you want to work on a specific issue
 
 1. locally run each of the tests marked with that issue in the expected failures file.
 
-    E.g.:
+   E.g.:
 
-    ```bash
-    make test-acceptance-from-core-api \
-    TEST_SERVER_URL=https://localhost:9200 \
-    TEST_OCIS=true \
-    TEST_WITH_GRAPH_API=true \
-    STORAGE_DRIVER=OCIS \
-    BEHAT_FEATURE='tests/acceptance/features/coreApiVersions/fileVersions.feature:147'
-    ```
+   ```bash
+   make test-acceptance-from-core-api \
+   TEST_SERVER_URL=https://localhost:9200 \
+   TEST_OCIS=true \
+   TEST_WITH_GRAPH_API=true \
+   STORAGE_DRIVER=OCIS \
+   BEHAT_FEATURE='tests/acceptance/features/coreApiVersions/fileVersions.feature:147'
```

Comment:
maybe we can also add a little bit of a hint or note saying that the line number might change because  
this might not work if we change something in feature files `tests/acceptance/features/coreApiVersions/fileVersions.feature:147` 

OR
`tests/acceptance/features/coreApiVersions/fileVersions.feature:<line-number-pointing-to-a-scenario>`

Reply:
Let it be as an example. I will write a note on running specific scenario using line number

Reply:
added here https://github.com/owncloud/ocis/blob/82e1423056d8d214a57b57a5ae2285561659a8e6/docs/ocis/development/testing.md?plain=1#L205